### PR TITLE
[sc-6667] Implement cancel subscription

### PR DESCRIPTION
**Pull request by operramon**

### CODE REVIEW

File: apps/dashboard/src/app/account/billing/subscriptions/subscriptions.component.ts [hunk 1]
```diff
@@ -7,7 +7,6 @@ import { AccountService, STFTrackingService } from '@flaps/core';
 import { COUNTRIES } from '../utils';
 import { Currency } from '../billing.models';
 import { WINDOW } from '@ng-web-apis/common';
-import { SubscribedAccountDeleteComponent } from '../../account-manage/account-delete/subscribed-account-delete.component';
 import { DeprecatedCalculatorComponent } from '../calculator/deprecated-calculator.component';
```
Comment (mpellerin42) [inline apps/dashboard/src/app/account/billing/subscriptions/subscriptions.component.ts:10]: Is this normal that we're still using a component marked as deprecated? `DeprecatedCalculatorComponent`

File: libs/common/src/assets/i18n/en.json [hunk 2]
```diff
@@ -197,21 +198,24 @@
...
-	"billing.quotas.answers": "Generative answer",
 	"billing.quotas.files": "Maximum number of files",
 	"billing.quotas.files-up-to": "Files up to",
+	"billing.quotas.generative": "Generative answer",
 	"billing.quotas.ingestion": "Ingestion",
 	"billing.quotas.kbs": "Number of knowledge boxes",
-	"billing.quotas.medias": "Video & audio indexing ",
-	"billing.quotas.paragraphs": "Number of paragraph",
-	"billing.quotas.paragraphs_stored": "Paragraph stored",
+	"billing.quotas.media": "Video & audio indexing ",
```
Comment (mpellerin42) [inline libs/common/src/assets/i18n/en.json:206]: I guess the extra space isn't in purpose?
```suggestion
	"billing.quotas.media": "Video & audio indexing",
```

**Review by mpellerin42 — APPROVED** [review]

File: libs/common/src/assets/i18n/en.json (hunk 2 above)
Comment (operramon) [reply libs/common/src/assets/i18n/en.json]: Fixed

File: apps/dashboard/src/app/account/billing/subscriptions/subscriptions.component.ts (hunk 1 above)
Comment (operramon) [reply apps/dashboard/src/app/account/billing/subscriptions/subscriptions.component.ts:10]: This deprecated component is not used any more. It was used for the old subscription plans. There's a pending task to remove it (sc-6568)

**Pull request merged**